### ISSUE TITLE
Dokapon Kingdom Support?

### ISSUE THREAD

**Issue by zegmin** [wilm0x42/wii-gc-adapter-inject#6]
Is there anyway you could test if this would support Dokapon Kingdom for Wii? The game natively supports GameCube controllers. If possible injecting the .gct into the .iso and then installing it as WiiU Wii VC would be optimal, as getting a disc of the game is hard given it costs $100-200. This would be amazing, it is quite honestly one the best party games of all time.

**Comment by wilm0x42** [comment]
Right now I intend for this hack to be specifically for Smash, since [Chadderz' implementation of the same idea](https://github.com/Chadderz121/wup-028-bslug) theoretically works without per-game configuration. Mine uses gecko codes and such for simplicity, but Chadderz uses his own modding tool/framework called BrainSlug, which allows hacks to apply to multiple different games.
Right now, I don't think BrainSlug supports anything other than real discs, and even then, his gamecube adapter code doesn't yet (assuming it ever will) support all IOSes.

tl;dr Unless all hope dies for [Chadderz' code](https://github.com/Chadderz121/wup-028-bslug) to support more games and launchers, this hack is only for Smash.
I may try to assist the broader wii/wiiu hacking community with this at some point, but not until I have Smash working properly.

Until then, I wish you the best of luck!
